notebooks/official/model_evaluation/custom_tabular_regression_model_evaluation.ipynb

```diff
@@ -989,9 +955,9 @@
       "source": [
         "#### Load the saved model\n",
         "\n",
-        "Your model is stored in a TensorFlow SavedModel format in a Cloud Storage bucket.  Now load it from the Cloud Storage bucket, and then you can perform tasks such as model evaluation and make prediction requests.\n",
+        "Your model is stored in a TensorFlow SavedModel format in a Cloud Storage bucket.  Now load it from the Cloud Storage bucket, and then you can run model evaluation and prepare it for prediction requests.\n",
```

Comment:
suggested edit:

- FROM: Now load it from the Cloud Storage bucket, and then you can run model evaluation and prepare it for prediction requests.

TO: Once you load the model from the Cloud Storage bucket you can run model evaluation and prepare it for prediction requests.